### ISSUE TITLE
have global candidate shifting

### ISSUE THREAD

**Issue by bboettcher3** [bboettcher3/gRainbow#61]
"Just thinking out loud here, maybe it would be useful to have global <> buttons/selectors to be able to switch the candidates on the fly, while performing (shifting maybe?)."
_Originally posted by @AtmanActive in https://github.com/bboettcher3/gRainbow/issues/60#issuecomment-1581019710_

**Comment by Grupletruge** [comment]
I feel like there are a lot of interesting things that could be done in this vein, I personally would love to see "Candidate Mapping" or perhaps "Generator mapping", I'll call it. Essentially to be able to assign a midi octave to each generator, that way i can grab higher notes as i move up the keyboard, or create a few "soundsets", 1 in each octave. I feel this would make the instrument much more expressive, as would other options to use midi to change candidates mid performance, etc.

**Comment by bboettcher3** [comment]
I like this idea! Another variation of it make a key mapping envelope for each generator so that sounds can crossfade between each other as you play different octaves

**Comment by Grupletruge** [comment]
Yeah that sounds awesome! Thanks for the response and for your work on the plugin, and for sharing it!
